web/themes/custom/pineapple/sass/components/navigation/_main-menu.scss

```diff
@@ -1,2 +1,119 @@
+.menu {
+  overflow: hidden;
+  background-color: $primary;
+  text-align: center;
 
+  .hamburger {
```

Comment:
Make sure the hamburger always appears on the right not left.

Reply:
Also increase the size of the hamburger, it's too small right now.

Reply:
Done.

Reply:
This is still not done the hamburger still appears on the left.